docs/wikidata-game.md

```diff
@@ -0,0 +1,30 @@
+# Introduction
+In order to measure the quality fo the pipeline, and to add references to Wikidata we implemented a version of the distributed game.
+
+The main page of the game platform can be found: [here](https://tools.wmflabs.org/wikidata-game/distributed/)
+
+The code for the platform that the games runs on can be found: [here](https://bitbucket.org/magnusmanske/wikiata-game/src/master/public_html/distributed/)
+
+The version of the game deployed in May 2020 by the Reference Island team can be found: [here](https://tools.wmflabs.org/wikidata-game/distributed/)
+
+# Hosting our Game
+The platform requires an api to be provided that serves it "tiles" that can them be shown to users. The api is also sent feedback about user actions.
+The api also provides a small summary for inclusion on the main page of the game.
+
+The API must be accessible by the platform over http(s). We therefore chose to host our game on toolforge.
+
+Documentation and the landing page for toolforge can be found: [here](https://tools.wmflabs.org/admin/)
+
+It is hosted under the tool name: `wd-ref-island`
```

Comment:
This is the point where we provide a contact email for readers to request to be added as tool maintainers.

Reply:
Added link to the toolforge summary page about our tool. Sadly this appears to currently temporarily believe our tool is unavailable. I added a comment to https://phabricator.wikimedia.org/T229261 so I hope it's fixed v soon.